src/components/Badge/Badge.tsx

```diff
@@ -6,38 +6,45 @@ import { Link } from "../Link";
 
 const polyRef = forwardRef as PolyRefFunction;
 
-export type BadgeType =
-  | "new"
-  | 'error'
-  | 'info'
-  | 'warning'
-  | 'success';
+export type BadgeType = "new" | "error" | "info" | "warning" | "success";
 
 export type BadgeProps = {
   icon?: string;
   className?: Argument;
   size?: "md" | "sm";
-  children: string;
+  children?: React.ReactNode;
```

Comment:
Why optional ?

Reply:
you are right, it's corrected in another commit